maven-slf4j-provider/src/main/java/org/slf4j/impl/MavenFailLevelLoggerState.java

```diff
@@ -19,29 +19,32 @@
  * under the License.
  */
 
-import org.slf4j.Logger;
+import org.slf4j.event.Level;
 
 /**
- * MavenSimpleLoggerFactory
+ * Responsible for keeping state of whether the threshold of the --fail-level flag has been hit.
  */
-public class MavenSimpleLoggerFactory
-    extends SimpleLoggerFactory
+class MavenFailLevelLoggerState
```

Comment:
Rename `MavenSimpleLoggerFactory` to `MavenFailLevelLoggerState`? That's weird, right?

Reply:
I don't see any Maven related code. How about renaming the class to `LogLevelRecorder`?

Reply:
I guess GitHub messed up the diff, I can't explain the renaming otherwise.
-- 
I like the name LogLevelRecorder, but currently it also holds state whether the threshold has been hit, so it might not fully describe it's intention.